api/src/test/resources/org/openmrs/api/include/EncounterServiceTest-initialData.xml

```diff
@@ -46,6 +46,8 @@
   <person person_id="3" gender="F" dead="false" creator="1" date_created="2005-01-01 00:00:00.0" voided="false" uuid="4b3f42da-2029-4e47-9396-a1b6a969e802"/>
   <person_name person_name_id="2" preferred="true" person_id="3" given_name="John3" middle_name=" " family_name="Doe" creator="1" date_created="2005-01-01 00:00:00.0" voided="false" uuid="38f9c631-7168-4944-8e2b-e96b1b0ee882"/>
   <patient_identifier patient_identifier_id="2" patient_id="3" creator="1" date_created="2005-01-01 00:00:00.0" identifier="12345" identifier_type="1" preferred="1" voided="false" location_id="1"  uuid="037d7208-da34-48e3-9761-24b614b99b56"/>
+  <patient_identifier patient_identifier_id="5" patient_id="2" creator="1" date_created="2022-04-26 00:00:00.0" identifier="4321" identifier_type="1" preferred="1" voided="false" location_id="1" uuid="727bdbf7-ecee-4cb0-88c5-99e7e9eea371"/>
+  <patient_identifier patient_identifier_id="6" patient_id="2" creator="1" date_created="2022-05-10 00:00:00.0" identifier="4321" identifier_type="1" preferred="1" voided="false" location_id="1" uuid="31572c4e-ab9a-4fbf-a007-598a88562cb6"/>
```

Comment:
Not sure why a second patient_identiifier was added that is basically the same as the previous one.  One other question, how does test return two encounters?  I don't see any encounters for patient_id=2 in the test XML data file?  Maybe I'm missing something.

Reply:
> Not sure why a second patient_identiifier was added that is basically the same as the previous one

I have removed one.

> One other question, how does test return two encounters? I don't see any encounters for patient_id=2 in the test XML data file?

Here is where am failing to figure out what exactly I need to do, When I add an encounter for patient_id=2 in the test XML data file, it causes other tests to fail. cc: @sjmckee